basyx/aas/examples/tutorial_aasx.py

```diff
@@ -23,10 +23,10 @@
 
 
 #########################################################################################
-# Step 1: Setting up an SupplementaryFileContainer and AAS & submodel with File objects #
+# Step 1: Setting up a SupplementaryFileContainer and AAS & submodel with File objects #
 #########################################################################################
```

Comment:
Can you adjust the length of the `#########...` lines above and below to match the length of the line you changed?